### PR TITLE
Resolve some warnings

### PR DESCRIPTION
**Pull request by carstene1ns**
Some random warnings I encountered while working with the codebase.

### CODE REVIEW

File: src/graphics/bmfont.cpp [hunk 1]
```diff
@@ -78,12 +78,12 @@ BMFont::~BMFont()
     cleanup();
 }
 
-const uint32_t BMFont::height()
+uint32_t BMFont::height()
```
Comment (isage) [inline src/graphics/bmfont.cpp]: Yes. But for correctness sake, this and others should be `uint32_t func const`

Comment (carstene1ns) [reply src/graphics/bmfont.cpp]: Sorry, I am more of a `C` guy. So moving this `const` in all functions is the way to go?

**Comment by isage** [comment]
Err, no, not just moving. const after function is used to tell compiler
that this function doesn't change object internal state. So in our case it
only applicable to getters.

**Comment by carstene1ns** [comment]
Sorry to say that, but "whatever works for you".

Thing is, this is the only  *active* nxengine fork, so redirecting stuff here is the only thing one could do.

**Comment by isage** [comment]
Looks good.

**Pull request merged**